README.md

```diff
@@ -123,14 +136,19 @@ module.exports = {
 
 ### `algorithm`
 
-Type: `String|Function`
+Type:
+
+```ts
+type algorithm = string | Function;
```

Comment:
Can we show `Function`, i.e. arguments and returned type?

Reply:
But this function also returns a function (function that compresses)  right? We would be able to add types for its params alone?

Reply:
Here types for this function https://github.com/webpack-contrib/compression-webpack-plugin/blob/master/src/index.js#L197

Reply:
https://github.com/webpack-contrib/compression-webpack-plugin/blob/2dd0f41cf8231247f5d9fa70df0c68011d510b56/types/index.d.ts#L172 I've added this type...